core/loader/json-feed.ts

```diff
@@ -91,19 +91,36 @@ function validate<ValidatedType>(
   return true
 }
 
+const IMG_URL_REGEXP = /<img[^>]+src="?([^"\s]+)"?\s*\/?>/g
```

Comment:
We can use `DOMParser` instead.

Developers can use `'` instead of `"`.

Reply:
Updated